package-depencies.txt

```diff
@@ -0,0 +1,5217 @@
+[INFO] Scanning for projects...
```

Comment:
also I do not understand why do we need this file in a repo?

Reply:
Oh that was added by mistake. Let me remove it.